### ISSUE TITLE
Green Screen Mario Kart DD

### ISSUE THREAD

**Issue by RamiLate80s** [emukidid/swiss-gc#473]
Ever since updating from swiss rev.990 to 1026, Mario Kart Double Dash won't start properly and I get a green screen.

**Comment by Extrems** [comment]
Can you give more details? I can't reproduce this.

**Comment by RamiLate80s** [comment]
Yes, I'm using a gc loader with 256gb sandisk sd. When I select the game and after the GC animation plays, the green screen appears. This only started happening on rev.1026. I cant think of anything else I did different and it's only this game for some reason

**Comment by Extrems** [comment]
Does the SD Card in your SD Card Adapter or SD2SP2 work for playing games?

**Comment by RamiLate80s** [comment]
Yes it works for playing games from the sd card adapter. I only use the Sd2sp2 for saving swiss settings and virtual memory card saves. The other games work perfectly fine as it did before.

**Comment by Extrems** [comment]
You're implicitly using it for patched disc fragments. Disc reads that hit executable files are redirected to your SD2SP2.

**Comment by RamiLate80s** [comment]
Im sorry I think I explained incorrectly. I use two sd cards. One in my gc loader for playing games and the other is in my sd2sp2 just for vmc saves and settings.

**Comment by Extrems** [comment]
You didn't, and what I said still holds when you're using your GC Loader.

**Comment by RamiLate80s** [comment]
I have not been able to figure out how to fix my issue. I do appreciate your time and help though, thank you.

**Comment by RamiLate80s** [comment]
This is a link to a video clip I recorded of the issue im having. If anyone can tell me what I need to do to fix this would greatly appreciate it! https://youtu.be/gY9ZGXKQP0Q
